app/services/proxy_status/tftp.rb

```diff
@@ -0,0 +1,22 @@
+module ProxyStatus
+  class TFTP < Base
+    def initialize(proxy, opts = {})
+      super(proxy, opts, 'TFTP')
```

Comment:
nitpick, can probably use self.humanized_name by default instead of override the initalizer.